src/lib/x509/x509_ext.h

```diff
@@ -661,10 +661,11 @@ class BOTAN_PUBLIC_API(3, 9) IPAddressBlocks final : public Certificate_Extensio
 
             IPAddressChoice() = default;
 
-            // NOLINTNEXTLINE(*-explicit-conversions) FIXME
-            IPAddressChoice(std::optional<std::span<const IPAddressOrRange<V>>> ranges);
-
          private:
+            friend class IPAddressBlocks;
+
+            explicit IPAddressChoice(std::optional<std::span<const IPAddressOrRange<V>>> ranges);
+
             std::optional<std::vector<IPAddressOrRange<V>>> m_ip_addr_ranges;
```

Comment:
Was there a reason this was marked as a fixme instead of putting the `explicit` there right away?

Reply:
Amazing Github changeset, I meant the removed `// NOLINTNEXTLINE(*-explicit-conversions) FIXME` above